src/main/java/com/abrus/Main.java

```diff
@@ -0,0 +1,33 @@
+package com.abrus;
+
+import com.abrus.user.model.UserRegistrationDto;
+import com.abrus.user.model.UserResponseDto;
+import com.abrus.user.repository.UserRepository;
+import com.abrus.user.repository.UserRepositoryImpl;
+import com.abrus.user.service.UserService;
+import com.abrus.user.service.UserServiceImpl;
+import com.abrus.user.validator.UserRegistrationValidator;
+import com.abrus.user.validator.UserRegistrationValidatorImpl;
+
+import java.util.List;
+import java.util.Optional;
+
+public class Main {
+
+    public static void main(String[] args) {
+        UserRepository userRepository = new UserRepositoryImpl();
+        UserRegistrationValidator userRegistrationValidator = new UserRegistrationValidatorImpl(userRepository);
```

Comment:
Not sure validator class should depend on repository
```suggestion
        UserRegistrationValidator userRegistrationValidator = new UserRegistrationValidatorImpl();
```